### ISSUE TITLE
Strange Data Loss Issue(0.13.1)

### ISSUE THREAD

**Issue by AlanDunaway** [olivierkes/manuskript#1028]
I got a strange data loss issue to report with 0.13.1.
I saved my project after editing the Summary in a few text files. When I loaded the project again, part of the edits was gone.
Anyone else had this issue?
Thanks.

**Comment by rotoclabs** [comment]
I've been encountering this issue pretty often and realized the issue happens when multiple instances of Manuskript is launched with same project. Many a times, I forget that I've had an instance open and re-launch the application and happily edit the project from the new one. If I so happen to close this latest edit first, and later the old running window of the software (which still has a partial edit that has started, and mostly this window goes unnoticed), and close this older window as the last one, then this overwrites all the good work that has happened in the previous window.

I think Manuskript does not acquire a lock on the file and warn that a window is already open with this project. That one change could solve this problem. I'm on 0.14.0 (Linux)

**Comment by TheJackiMonster** [comment]
Yeah, launching the application multiple times on the same project is an issue, I assume.
